recipes/multiqc-plugins/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "multiqc-plugins" %}
```

Comment:
Would it be possible to use a more specific (unique) name, such as "zavolan_multiqc_plugins"?

Reply:
Sure. I just renamed it.